examples/browser_example.html

```diff
@@ -98,11 +98,11 @@ <h1>osdatahub Examples</h1>
   );
   new L.Control.Zoom({ position: "bottomright" }).addTo(map);
   const tileLayer = L.tileLayer(
-    "https://tile.openstreetmap.org/{z}/{x}/{y}.png",
+    "https://api.os.uk/maps/raster/v1/zxy/Light_3857/{z}/{x}/{y}.png?key=sks1ArFMuYctiVGbAmt8GtRSIuFeCV2Y",
     {
       maxZoom: 19,
       attribution:
-        '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>',
+        'Contains OS data &copy Crown copyright and database rights 2022',
```

Comment:
Just a thought since we are near the end of the year... could the year be added dynamically into the string e.g. something like `new Date().getFullYear()`?

Reply:
Does putting YYYY into the attribution do this automatically, or did I dream that? :D

Reply:
@JEPooley Just tried it and it doesn't seem to work! I made the change as @FHunt-OS suggested